deploy-agent/deployd/agent.py

```diff
@@ -111,8 +112,12 @@ def _send_deploy_status_stats(self, deploy_report):
             tags['stage_name'] = self._response.deployGoal.stageName
         if deploy_report.status_code:
             tags['status_code'] = deploy_report.status_code
-        if deploy_report.output_msg and deploy_report.output_msg.find("teletraan_config_manager") != -1:
-            tags['error_source'] = DeployErrorSource.TELEFIG
+        if deploy_report.output_msg: 
+            if deploy_report.output_msg.find("teletraan_config_manager") != -1:
+                tags['error_source'] = DeployErrorSource.TELEFIG
+            if check_telefig_unavaialble_error(deploy_report.output_msg):
```

Comment:
maybe we could do "else" so the string compare does not happen twice which might be expensive if the output msg is large.  maybe check "unavailable error" first , else check others.

Reply:
yeah true, output msg can be large